### PR TITLE
Switch to using react-helmet

### PR DESCRIPTION
**Pull request by cmcewen**
`react-document-meta` is unsupported and I was having some issues with caching headers on SSR - this switches to using `react-helmet`

### CODE REVIEW

File: src/server/html.js [hunk 1]
```diff
@@ -13,10 +14,20 @@ export default (config, assets, store, headers, component) => {
...
-      <Body assets={assets} store={store} headers={headers} component={component} />
+  const header = ReactDOM.renderToString(<Head additions={config.html.head} assets={assets} store={store} headers={headers} />);
+  const body = ReactDOM.renderToString(<Body assets={assets} store={store} headers={headers} component={component} />);
+  const head = Helmet.rewind();
+
+  return `
+    <!doctype html>
+    <html ${head.htmlAttributes.toString()}>
+        <head>
+            ${header.replace('<head>', '').replace('</head>', '')}
```
Comment (bdefore) [inline src/server/html.js:25]: i'm unfamiliar with `${ ... }` within JSX, is this doing something that `{ ... }` does not do?

Comment (cmcewen) [reply src/server/html.js:25]: This is using a template literal (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) - was just copying the example https://github.com/nfl/react-helmet#as-string-output

Comment (bdefore) [reply src/server/html.js:25]: oh, silly me. i missed the back ticks around what's being returned. thanks.

**Comment by bdefore** [comment]
@cmcewen This looks good, thanks. I'm going to investigate how this impacts the universal-redux-starter and universal-redux-jwt projects to determine if it deems a major version bump.

**Comment by bdefore** [comment]
@cmcewen everything looks fine and i've made a 4.1.0 release that lands these changes

**Comment by cmcewen** [comment]
💯  awesome thanks!

**Pull request merged**